dev_tools/packaging/verify-published-package.sh

```diff
@@ -28,7 +28,7 @@
 ################################################################################
 
 set -e
-trap "{ echo -e '\033[31mFAILED\033[0m'; }" ERR
+trap "{ echo -e 'FAILED'; }" ERR
```

Comment:
There was nothing wrong with the echo statement.  What I meant was that when I run `source dev_tools/packaging/set-dev-version.sh` manually then later I run a command that returns a non-zero exit command, my terminal exits.  Is there some way to indicate that this script shouldn't be run by a developer or make it not do this?

Reply:
Oh not the trap. the "set -e".  can add "set +e".

Reply:
That will help but if a command in this script fails my terminal would still exit.

Reply:
Uh. it didn't in my testing.  .

Reply:
oh sorry forgot to add to script.  this worked for me